doc/source/whatsnew/v0.23.0.txt

```diff
@@ -272,6 +272,7 @@ Other API Changes
 - :class:`IntervalIndex` and ``IntervalDtype`` no longer support categorical, object, and string subtypes (:issue:`19016`)
 - The default ``Timedelta`` constructor now accepts an ``ISO 8601 Duration`` string as an argument (:issue:`19040`)
 - ``IntervalDtype`` now returns ``True`` when compared against ``'interval'`` regardless of subtype, and ``IntervalDtype.name`` now returns ``'interval'`` regardless of subtype (:issue:`18980`)
+- ``KeyError`` now raises instead of ``ValueError`` when using :meth:`drop()` to remove a non-existent element in an axis of ``Series``, ``Index``, ``DataFrame`` and ``Panel`` (:issue:`19186`)
```

Comment:
can you re-word, start with ``:meth:`drop()` `` (side issue does this render?) you may hve to say ``:meth:`DataFrame.drop` `` (and prob list for all of them Index, Series, Panel).

Reply:
No, I don't think that'll render / link. I usually do

    :meth:`~DataFrame.drop`

When I want to refer to a generic `.drop` method, which renders as `.drop()`, but links to DataFrame.